client/auth.go

```diff
@@ -2,37 +2,41 @@ package client
 
 import (
 	"bytes"
-	"cloud.google.com/go/firestore"
 	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
-	"golang.org/x/oauth2"
-	"google.golang.org/api/option"
 	"io/ioutil"
 	"log"
 	"os"
 	"os/user"
 	"path"
 	"time"
+
+	"cloud.google.com/go/firestore"
+	"golang.org/x/oauth2"
+	"google.golang.org/api/option"
 )
 
 func (c *fuzzitClient) ReAuthenticate(force bool) error {
 	usr, err := user.Current()
 	if err != nil {
 		log.Fatal(err)
 	}
-	CacheFile := path.Join(usr.HomeDir, ".fuzzit.cache")
+	cacheFile := path.Join(usr.HomeDir, ".fuzzit.cache")
 
 	if !force {
-		file, err := os.Open(CacheFile)
+		file, err := os.Open(cacheFile)
 		if err != nil {
 			return err
 		}
-		defer file.Close()
```

Comment:
Why this line is removed?

Reply:
It's moved after json.NewDecoder() because on e.g. windows one cannot remove opened file so we have to call file.Close() before os.Remove() and defer file.Close() means it would be called after.

Reply:
Got it. nice!